sql-cli/sql_cli/__main__.py

```diff
@@ -229,5 +256,41 @@ def init(
     rprint("Initialized an Astro SQL project at", project.directory)
 
 
+def _generate_dag(project: Project, workflow_name: str, generate_tasks: bool) -> Path:
```

Comment:
Is it possible to move this method to some other module. I think @tatiana 's idea was to keep only commands in this __main__.py file and we keep it clean by keeping implementation methods in other corresponding modules.

Reply:
Let's do that as part of the newly created ticket #1235